sdk/rust/src/svm.rs

```diff
@@ -129,9 +172,7 @@ impl Svm {
             .map_err(|_| {
                 ClientError::SvmError("Failed to fetch express relay metadata".to_string())
             })?;
-        deserialize_metadata(data).map_err(|_| {
-            ClientError::SvmError("Failed to deserialize express relay metadata".to_string())
-        })
+        deserialize_metadata(data)
```

Comment:
since this is the only place you call this method, should we inline it the code here?

not sure searchers should ever call `deserialize_metadata`